test/pylib_test/test_manager_client_call.py

```
@@ -0,0 +1,204 @@
+    assert cluster.entered.is_set(), "operation never started"
+    assert len(client._manager.tasks_history) == 1, "operation was not left for the drain"
+    op, descr = next(iter(client._manager.tasks_history.items()))
+    assert not op.done()
+    assert descr.startswith("server_stop")
+
+    # Once it completes it removes itself, which is what leaves after_test()'s
+    # drain looking only at genuinely in-flight work.
+    cluster.release()
+    async with asyncio.timeout(TIMEOUT):
+        while client._manager.tasks_history:
+            await asyncio.sleep(0.1)
```

Ruff (0.16.1)

[warning] Use `asyncio.Event` instead of awaiting `asyncio.sleep` in a `while` loop

(ASYNC110)